From 7baa0411acc255d42c1c3bec02ec028ee6d06f28 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garrett.brown@aclima.io>
Date: Tue, 9 Aug 2022 18:14:22 -0700
Subject: [PATCH] Update ament to "humble" branches

---
 ros2.repos | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/ros2.repos b/ros2.repos
index 7df52b7..e80d864 100644
--- a/ros2.repos
+++ b/ros2.repos
@@ -2,31 +2,31 @@ repositories:
   ament/ament_cmake:
     type: git
     url: https://github.com/ament/ament_cmake.git
-    version: galactic
+    version: humble
   ament/ament_index:
     type: git
     url: https://github.com/ament/ament_index.git
-    version: galactic
+    version: humble
   ament/ament_lint:
     type: git
     url: https://github.com/ament/ament_lint.git
-    version: galactic
+    version: humble
   ament/ament_package:
     type: git
     url: https://github.com/ament/ament_package.git
-    version: galactic
+    version: humble
   ament/google_benchmark_vendor:
     type: git
     url: https://github.com/ament/google_benchmark_vendor.git
-    version: galactic
+    version: humble
   ament/googletest:
     type: git
     url: https://github.com/ament/googletest.git
-    version: galactic
+    version: humble
   ament/uncrustify_vendor:
     type: git
     url: https://github.com/ament/uncrustify_vendor.git
-    version: galactic
+    version: humble
   eProsima/Fast-CDR:
     type: git
     url: https://github.com/eProsima/Fast-CDR.git
-- 
2.34.1

