From 748a74fd6480998807607284f93ac534cab9d627 Mon Sep 17 00:00:00 2001
From: Garrett Brown <eigendebugger@gmail.com>
Date: Fri, 12 Aug 2022 16:48:41 -0700
Subject: [PATCH] galactic: Update image_common to "rolling" branch

---
 ros2.repos | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ros2.repos b/ros2.repos
index 707ed7a..42add9f 100644
--- a/ros2.repos
+++ b/ros2.repos
@@ -58,7 +58,7 @@ repositories:
   ros-perception/image_common:
     type: git
     url: https://github.com/ros-perception/image_common.git
-    version: galactic
+    version: rolling
   ros-perception/laser_geometry:
     type: git
     url: https://github.com/ros-perception/laser_geometry.git
-- 
2.34.1

