From 893a4060608a7534efd0d98938fb3e9fc40b24ee Mon Sep 17 00:00:00 2001
From: juztamau5 <juztamau5@gmail.com>
Date: Thu, 3 Nov 2022 03:52:27 -0700
Subject: [PATCH] Disable Python bindings by default

---
 cv_bridge/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cv_bridge/CMakeLists.txt b/cv_bridge/CMakeLists.txt
index 9526862..75131c5 100644
--- a/cv_bridge/CMakeLists.txt
+++ b/cv_bridge/CMakeLists.txt
@@ -12,7 +12,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra)
 endif()
 
-option(CV_BRIDGE_DISABLE_PYTHON "Disable building Python bindings" OFF)
+option(CV_BRIDGE_DISABLE_PYTHON "Disable building Python bindings" ON)
 
 if(ANDROID)
   set(CV_BRIDGE_DISABLE_PYTHON ON)
-- 
2.34.1

