From 43c753e6c3df4ef1355359f86807147ccea00994 Mon Sep 17 00:00:00 2001
From: Garrett Brown <eigendebugger@gmail.com>
Date: Tue, 12 Nov 2024 14:27:28 -0800
Subject: [PATCH 2/7] Fix parallel make

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 37f48ac..e1fc377 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -115,6 +115,9 @@ include/Settings.h)
 
 add_subdirectory(Thirdparty/g2o)
 
+# Ensure g2o is built before linking ORB_SLAM3
+add_dependencies(${PROJECT_NAME} g2o)
+
 target_link_libraries(${PROJECT_NAME}
 ${OpenCV_LIBS}
 ${EIGEN3_LIBS}
-- 
2.43.0

