From 96ba665c29ee2ef827f93d81a1ee5bf5a3d1b330 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garrett.brown@aclima.earth>
Date: Tue, 1 Apr 2025 16:16:56 -0700
Subject: [PATCH 2/2] Force serial build

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7c3b2e5..f46bdee 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,7 @@ endif()
 string(TOLOWER ${CMAKE_BUILD_TYPE} MESON_BUILD_TYPE)
 
 if(NOT NJOBS)
-  set(NJOBS 0)
+  set(NJOBS 1)
 endif()
 
 find_package(OpenSSL REQUIRED)
-- 
2.43.0

