From da3f8a658342a6c36c327b3130d3b741ab24bb08 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 4 Jun 2021 20:12:07 -0700
Subject: [PATCH 2/3] [temp] Wayland: Disable window decorations

---
 xbmc/windowing/wayland/WindowDecorator.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/xbmc/windowing/wayland/WindowDecorator.cpp b/xbmc/windowing/wayland/WindowDecorator.cpp
index a0ae2ea2e4..3e0dfb8009 100644
--- a/xbmc/windowing/wayland/WindowDecorator.cpp
+++ b/xbmc/windowing/wayland/WindowDecorator.cpp
@@ -684,7 +684,8 @@ bool CWindowDecorator::IsDecorationActive() const
 bool CWindowDecorator::StateHasWindowDecorations(IShellSurface::StateBitset state) const
 {
   // No decorations possible if subcompositor not available
-  return m_subcompositor && !state.test(IShellSurface::STATE_FULLSCREEN);
+  //! @todo
+  return false;
 }
 
 CSizeInt CWindowDecorator::CalculateMainSurfaceSize(CSizeInt size, IShellSurface::StateBitset state) const
-- 
2.43.0

