From 19dcc1bff37118ad4d3fdcf42f5e2843a25be8bc Mon Sep 17 00:00:00 2001
From: Garrett Brown <eigendebugger@gmail.com>
Date: Wed, 3 Dec 2025 16:09:25 -0800
Subject: [PATCH 4/4] Increase publisher QoS depth to 10 for improved WiFi
 streaming

---
 src/CameraNode.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CameraNode.cpp b/src/CameraNode.cpp
index 91b9645..507ce96 100644
--- a/src/CameraNode.cpp
+++ b/src/CameraNode.cpp
@@ -170,7 +170,7 @@ create_camera_publisher(rclcpp::Node *node, const std::string &topic)
   image_transport::CameraPublisher pub_camera;
 
   // Publisher QoS
-  const rclcpp::QoS publish_qos {1};
+  const rclcpp::QoS publish_qos {10};
 
   // NOTE: ROS 2 Rolling deprecates image_transport::create_camera_publisher(Node*, ...),
   // but this node still uses the older API. We intentionally call the deprecated
-- 
2.43.0

